internal/hcsoci/resources_wcow.go

```diff
@@ -171,6 +175,36 @@ func setupMounts(ctx context.Context, coi *createOptionsInternal, r *resources.R
 					return errors.Wrapf(err, "adding SCSI EVD mount failed %+v", mount)
 				}
 				r.Add(scsiMount)
+			} else if strings.HasPrefix(mount.Source, "sandbox://") {
```

Comment:
Can we use a mount-type instead of checking a prefix?

Reply:
Yea I'd asked that above. Kathryn's suggestion seems to be no, but it fits more in line with what we have for the others. We currently set the type to bind for these for Linux, should we just check for bind here or for Windows name it "sandbox" or similar as the type is irrelevant for OCI on Windows and is just for us really

Reply:
I think setting the type to `sandbox` is better since bind mounts is not really a term we use in WCOW. But since Kathryn thinks otherwise we need a tie breaker here. :P Maybe @kevpar can comment?

Reply:
😆 Fair enough

Reply:
lol I see what you mean now. In the LCOW situation, we need the mount type to be bind for when the mount gets to runc to parse and actually do the mount into the container. I'm fine with using a mount type here instead of checking for a prefix string, but we'd still need to check for the prefix string in cri and we'd need to make sure that the mount type is set to bind in the LCOW situation when we're done handling it.

Reply:
I think if the platform is Windows vs Linux we can assign "sandbox" or "bind" depending on which it is, does that sound good?

Reply:
I'm fine with it. We should double check to see if we need to support both the prefix and any new mount type though for a time.

Reply:
Should just be the mount type we have to support, no ones using this (if it not being checked in was any clue 😂). So change this to check for type "sandbox" for Windows? All in agreeance, going once, going twice..

Reply:
I'll make the corresponding change on cri also very shortly

Reply:
I've thought about this more. I think we should be consistent with how lcow does it, so the prefix route.

Reply:
Sweet! I'll open up the cri PR, are we good merging this then?

Reply:
Oop nvm I misread your comment/forgot how this works as its been awhile. I'll change back to the prefix approach

Reply:
@katiewasnothere Done!